### PR TITLE
Fixes #1432: Refactoring the code of StatusService

### LINKED ISSUE #1432: Refactor StatusService.java
- Issue type: Feature request

### Short description
Make changes as per the discussion in 
https://github.com/loklak/loklak_server/pull/1428#discussion_r132326072

I'll work on this.

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #1432 

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@vibhcool @singhpratyush @djmgit @Achint08 @hemantjadon @SKrPl Please review.

File: src/org/loklak/api/admin/StatusService.java [hunk 1]
```diff
@@ -74,34 +74,8 @@ public static JSONObject status(final String protocolhostportstub) throws IOExce
...
-        long local_messages = DAO.countLocalMessages();
-        long local_users = DAO.countLocalUsers();
-
-        post.setResponse(response, "application/javascript");
-
-        // generate json
-        Runtime runtime = Runtime.getRuntime();
-        JSONObject json = new JSONObject(true);
-        JSONObject system = new JSONObject(true);
+    public static JSONObject system(Runtime runtime) throws Exception {
```
Comment (singhpratyush) [inline src/org/loklak/api/admin/StatusService.java]: Method name can be more clear, e.g. `getSystemConfig`. Also for other methods in the diff.

**Comment by kavithaenair** [comment]
@singhpratyush Please re-review.
Output:
image

File: src/org/loklak/api/admin/StatusService.java (hunk 1 above)
Comment (kavithaenair) [inline src/org/loklak/api/admin/StatusService.java]: Made changes.

**Comment by kavithaenair** [comment]
@singhpratyush index and message are interdependent in this.

**Comment by singhpratyush** [comment]
@kavithaenair: I was talking about this part - https://github.com/kavithaenair/loklak_server/blob/25e980e86b9992b826d99be87a623b37a22e7870/src/org/loklak/api/admin/StatusService.java#L201L283.

It can be moved to a separate function.

**Comment by kavithaenair** [comment]
@singhpratyush Its done

**Pull request merged**